src/popup/modules/QrLib/QrError.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Errors in QR code generation
+ *
+ * @module QrLib/QrErrors
+ */
+
+export class DataOverflowError extends Error {
+    constructor(...rparams) {
```

Comment:
```suggestion
    constructor(message, ...rparams) {
```

Better start like this...